### PR TITLE
Changes for documentations

### PR DESCRIPTION
**Pull request by Xiaoxuan1121**
Might need a link for 'glmtnet-package' in functions cv.sparsegl() and plot.cv.sparsegl().

### CODE REVIEW

File: R/norms.R [hunk 1]
```diff
@@ -1,11 +1,21 @@
-two_norm <- function(x) sqrt(sum(x^2))
+# Returns L^1-norm (Absolute-value norm).
```
Comment (dajmcdon) [inline R/norms.R]: For all of these, can you add the Roxygen and export? Let's document them all together. See [here](https://r-pkgs.org/man.html#multiple-man).

File: R/plot.cv.sparsegl.R [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 #' A plot is produced.
 #'
-#' @param x fitted \code{\link{cv.sparsegl}} object
-#' @param sign.lambda either plot against \code{log(lambda)} (default) or its
-#' negative if \code{sign.lambda=-1}.
-#' @param \dots other graphical parameters to plot
+#' @param x Fitted \code{\link{cv.sparsegl}} object
+#' @param sign.lambda Either plot against \code{log(lambda)} (default) or its
+#' negative if \code{sign.lambda = -1}.
+#' @param \dots Other graphical parameters to plot
```
Comment (dajmcdon) [inline R/plot.cv.sparsegl.R]: I think \dots is ignored. Is it? If so, please say so.

File: R/plot.sparsegl.R [hunk 3]
```diff
@@ -4,17 +4,28 @@
...
 #'   or the group norm.
-#' @param x_axis variable on the x-axis. Either the (log)-lambda
+#' @param x_axis Variable on the x-axis. Either the (log)-lambda
 #'   sequence (default) or value of the penalty. The penalty is scaled by its
 #'   maximum along the path.
-#' @param add_legend show the legend. Often, with many groups/predictors, this
+#' @param add_legend Show the legend. Often, with many groups/predictors, this
 #'   can become overwhelming.
-#' @param \dots other graphical parameters to plot
+#' @param \dots Other graphical parameters to plot.
```
Comment (dajmcdon) [inline R/plot.sparsegl.R]: Same as on `plot.cv.sparsegl`

File: _pkgdown.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  desc: >
+    Plot and predict for sparsegl or cv.sparsegl objects.
+- contents:
+    - plot.sparsegl
+    - coef.sparsegl
+    - predict.sparsegl
+    - print.sparsegl
+    - plot.cv.sparsegl
+    - coef.cv.sparsegl
+    - predict.cv.sparsegl
```
Comment (dajmcdon) [inline _pkgdown.yml]: replace with
 - starts_with(plot)
 - starts_with(predict)
 - starts_with(print)
 - starts_with(coef)

**Review by dajmcdon — CHANGES_REQUESTED** [review]

**Comment by Xiaoxuan1121** [comment]
Sure! I'll fix them up before making another PR!

File: R/norms.R [hunk 5]
```diff
@@ -1,21 +1,43 @@
-# Returns L^1-norm (Absolute-value norm).
+#' @describeIn grouped_sp_norm l1-norm (Absolute-value norm).
```
Comment (dajmcdon) [inline R/norms.R:1]: All of these need to be exported.

**Review by dajmcdon — CHANGES_REQUESTED** [review]

**Review by dajmcdon — APPROVED** [review]

**Pull request merged**